scripts/generate-sitemap.js

```diff
@@ -1,3 +1,4 @@
+const { data } = require('autoprefixer')
```

Comment:
I think this line is not required?

Reply:
This is because of auto import in VSCode. Removed.